pkg/kubelet/metrics/testdata/image_pull_duration_metric

```diff
@@ -0,0 +1,142 @@
+# HELP kubelet_image_pull_duration_seconds [ALPHA] 
+# TYPE kubelet_image_pull_duration_seconds histogram
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="0.5"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="1"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="2"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="3"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="4"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="5"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="6"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="8"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="10"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="20"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="30"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="45"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="60"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="120"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="180"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="240"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="300"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="360"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="480"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="600"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="900"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="1200"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="1800"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="2700"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="3600"} 0
+kubelet_image_pull_duration_seconds_bucket{image_size_in_bytes="GT100GB",le="Inf"} 1
```

Comment:
if you have a datapoint which is `GT100GB` shouldn't it also be `GT60GB`, `GT40GB`, etc?

Reply:
no each datapoint is only supposed to have one label, so GT60GB here means "Greater than 60GB but less than 100GB"

I can change the naming of the labels to make this more obvious. "GT60GB" -> "60GBB-100GB", does this look good?

Reply:
Yes, that change makes sense to me. `GT` presumably stands for "greater than" which basically means what I interpreted initially.

Reply:
yep, makes sense. Updated. PTAL